app/controllers/users_controller.rb

```diff
@@ -3,8 +3,20 @@ class UsersController < ApplicationController
 
   def downgrade
     flash[:alert] = "You have cancelled your premium subscription #{current_user.name}. We are sorry to see you go."
+    remove_private_wikis
     current_user.standard!
     redirect_to root_path
   end
 
+  private
+
+  def remove_private_wikis
+    current_user.wikis.each do |wiki|
+      if wiki.private == true
+        wiki.private = false
```

Comment:
you can use #update_attribute to make this one line, but not necessary! It is a cool method.